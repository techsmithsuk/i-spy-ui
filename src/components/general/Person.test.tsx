import React from 'react';
import { Person } from './Person';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';

describe("Person", () => {
    it("Should render without errors", () => {
        const component = renderer.create(
            <Person/>
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

test('renders person card', () => {
    const person = document.createElement('Person');
    ReactDOM.render(<Person />, person);
  });