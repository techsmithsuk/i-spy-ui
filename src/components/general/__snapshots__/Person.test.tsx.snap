// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person Should render without errors 1`] = `
<div
  className="personCard"
>
  <img
    alt="Image of Suspect"
    src="https://images.unsplash.com/photo-1555443805-658637491dd4?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=632&q=80"
  />
  SUSPECT NAME
</div>
`;
